apps/consoledot-rhosak/src/routes/data-plane/routes/DataPlaneTopicHeaderConnected.tsx

```diff
@@ -56,10 +62,10 @@ export const DataPlaneTopicHeaderConnected: VoidFunctionComponent<
         toggleExpanded(true);
       }}
       canOpenConnection={
-        instance ? ReadyStatuses.includes(instance?.status) : false
+        instance ? !SuspendedStatuses.includes(instance?.status) : false
```

Comment:
All these includes are logic that is spread across the app. We should have a single function that contains this logic.